src/Explorer/Panes/SettingsPane/Tooltip/index.tsx

```diff
@@ -0,0 +1,22 @@
+import { useId } from "@uifabric/react-hooks";
+import { ITooltipHostStyles, TooltipHost } from "office-ui-fabric-react/lib/Tooltip";
```

Comment:
This seems to be the same as in #578. If so, can we move this to a common location and avoid duplication?

Reply:
move in common folder.